apps/prairielearn/src/question-servers/freeform.ts

```diff
@@ -438,6 +438,7 @@ function checkData(data: Record<string, any>, origData: Record<string, any>, pha
   // The following code is deliberately formatted as it is to aid in comprehension,
   // so we prevent Prettier from reformatting the code to span multiple lines.
   // prettier-ignore
+  /* eslint-disable @stylistic/no-multi-spaces */
```

Comment:
There is already a prettier-ignore above this, which means there is overlap between the two different rules. Is that a concern? Is there something that eslint-stylistic brings that prettier doesn't? If so, is there a way to focus only on the rules that aren't covered by prettier?

Reply:
Totally agree, I think the question is if I should disable all conflicting rules, or try to get the config to line up with prettier.